File: LeetCode_Marathons/LCM5/Easy/Missing_Number/a.java
class Solution {
    public int missingNumber(int[] nums) {
        int needSumOfNums = 0, sumOfNums = 0;
        int n = nums.length;

        needSumOfNums = n * (n + 1) / 2;
        for (int i = 0; i < n; i++) sumOfNums += nums[i];

        return needSumOfNums - sumOfNums;
    }
}

File: LeetCode_Marathons/LCM5/Easy/Missing_Number/a.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        needSumOfNums, sumOfNums = 0, 0
        n = len(nums)

        needSumOfNums = n * (n + 1) // 2
        sumOfNums = sum(nums)

        return needSumOfNums - sumOfNums